README_Tutorials.md

```diff
@@ -33,7 +33,7 @@ Join the [Qiskit](https://ibm.co/joinqiskitslack) `#metal` slack channel to requ
 | Jan 13, 2022 | Forum | Q&A - The Qiskit Metal team will take any question |
 | Jan 27, 2022 | Forum | Helping users generate a custom component.
 | Feb 10, 2022 | Tutorial | <ul><li>[GUI icons](https://youtu.be/aE2Dsc67S8w)</li><li>[Jaynes Cummings model](https://youtu.be/dtwL_K-TJtY)</li><li>[template_options, options, default_options](https://youtu.be/qKLyEyW7cbQ)|
-| Feb 24, 2022 | Tutorial | <ul><li>[Will NOT be recorded.  Flip chip by guest speaker.]|
-| Mar 10, 2022 | TBD |
+| Feb 24, 2022 | Tutorial | [Flip chip by guest speaker, Sandoko Kosen.](https://youtu.be/BUojR6Uti5Q)|
+| Mar 10, 2022 | Forum| Metal community provide feedback. Things you want to have improved.|
```

Comment:
Maybe just

"Metal community ~~provide~~ feedback.

Reply:
Sure, it is updated now.